selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/RenameWorkspaceTest.java

```diff
@@ -55,7 +51,7 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    workspaceServiceClient.delete(CHANGE_WORKSPACE_NAME, user.getName());
+    workspaceServiceClient.delete(ws.getName(), user.getName());
```

Comment:
*ws* is being removed automatically at the pre-destroy step.
We need to try removal of workspaces with name **MIN_WORKSPACE_NAME** and **MAX_WORKSPACE_NAME** which could probably remain after the test execution in case of failure.